chainer/training/updater.py

```diff
@@ -289,8 +289,8 @@ def update_core(self):
             in_arrays_list[key] = self.converter(
                 batch[i::n], self._devices[key])
 
-        for model in six.itervalues(self._models):
-            model.cleargrads()
+        # for model in six.itervalues(self._models):
```

Comment:
My mistake, this should not have been committed here. I wrote my remark here: https://github.com/pfnet/chainer/pull/1551#discussion_r77295028